website/docs/cloud-docs/api-docs/policies.mdx

```diff
@@ -274,14 +347,15 @@ This PATCH endpoint requires a JSON object with the following properties as a re
+| `data.attributes.name`           | string         | (Current name)                                | Ignored if present.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `data.attributes.description`    | string         | `null`                                        | A description of the policy's purpose. This field supports Markdown and will be rendered in the Terraform Cloud UI.                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `data.attributes.query`          | string         |                                               | The OPA query to run. This is only valid for OPA policies                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `data.attributes.enforce`        | array\[object] |                                               | An array of enforcement configurations which map Sentinel file paths to their enforcement modes. Currently policies only support a single file, so this array will consist of a single element. The value provided **replaces** the enforcement map. To make an incremental update, you can first fetch the current value of this map from the [show endpoint](#show-a-policy) and modify it. If the path in the enforcement map does not match the Sentinel policy (`<NAME>.sentinel`), then the default `hard-mandatory` will be used. |
```

Comment:
```suggestion
| `data.attributes.enforce`        | array\[object] |                                               | An array of enforcement configurations that map Sentinel file paths to their enforcement modes. Policies support a single file, so this array should consist of a single element. Only valid for Sentinel policies. If the path in the enforcement map does not match the Sentinel policy (`<NAME>.sentinel`), then Terraform Cloud uses the default `hard-mandatory` enforcement level. |
```

Reply:
Valid for both sentinel and opa. fixed the docs to reflect the same